README.md

```diff
@@ -6,76 +6,70 @@
 
 mc-components aims to provide primitive components to build user interfaces in MasterClass.
```

Comment:
What are "primitive components?"

Reply:
you tell me

https://github.com/yankaindustries/mc-components/blame/87283a737629a9735f270a0ff4672739175b5b1c/README.md#L8